Python/pywarpx/WarpX.py

```diff
@@ -32,6 +33,7 @@ class WarpX(Bucket):
 
     def create_argv_list(self):
         argv = []
+        warpx.reduced_diags_names = reduced_diagnostics._reduced_diagnostics_dict.keys()
```

Comment:
This should follow the pattern of how it is done with diagnostics.

Reply:
The reason for the distinction is the way the reduced diagnostics are implemented in WarpX.  As opposed to regular diagnostics, indicated in the deck with `diagnostics.diags_names = <diag 1> <diag 2> ...`, the reduced diagnostics are called by a line of the form`warpx.reduced_diags_names = <reduced diag 1> <reduced diag 2> ...`

Reply:
Ok, I see. Though, I think that this really needs to be changed and the input argument should be `diagnostics.reduced_diags_names` but that would be a separate PR.

A way to make this cleaner perhaps would be to put this line in the `ReducedDiagnostic.initialize_inputs` method after adding the name to `WarpX.reduced_diagnostics._reduced_diagnostics_dict`.